src/mail/editor/SendMailModel.ts

```diff
@@ -420,7 +421,7 @@ export class SendMailModel {
 		const recipientsFilter = (recipientList: Array<PartialRecipient>) =>
 			deduplicate(
 				recipientList.filter((r) => isMailAddress(r.address, false)),
-				(a, b) => a.address === b.address,
+				(a, b) => cleanMailAddress(a.address) === cleanMailAddress(b.address),
```

Comment:
this is maybe too much in N^2, should we make a Set?

Reply:
set might be difficult because we need a custom compare, but we could map the mail addresses before going into the n² part.